src/vni-manager.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * File:  vni-manager.js
+ */
+var _ = require('underscore');
+
+var inputStates = require('./input-states');
+
+var IIP_VNID = '';
+
+/**
+ * Manage the node's vni's by vnid.  If no vnid is provided the IIP VNID will be used.
+ * If the vni for the specified vnid does not exist, a new one will be created.
+ *
+ * @this the component node instance unless otherwise specified on an API
```

Comment:
Did not see this before my last commit.  So if I understand write, you want the following mappings:
nodeInstance -> node
facade -> componentFacade.  

If so, will do that now.